### PR TITLE
[docs] [YB-TServer] Docs for ssl_protocols YB-TSever flag 

### LINKED ISSUE #13749: [Docs] Document ssl_protocols gflag
### Description

The flag ssl_protcols is not documented at https://docs.yugabyte.com/preview/reference/configuration/yb-tserver/ though it is covered in platform docs at https://docs.yugabyte.com/preview/yugabyte-platform/security/enable-encryption-in-transit/#enforcing-tls-versions

https://www.yugabyte.com/blog/yugabytedb-server-to-server-encryption/#:~:text=certificate.%20%5B1%5D-,Min%20SSL%20version,-Organizations%20may%20wish explains what this flag does.

### PR DESCRIPTION
**Pull request by aishwarya24**
resolves #13749 
Changes only in preview currently . will apply to other versions once its reviewed.
Build preview: https://deploy-preview-14325--infallible-bardeen-164bc9.netlify.app/preview/reference/configuration/yb-tserver/#ssl-protocols

### CODE REVIEW

**Review by es1024 — APPROVED** [review]

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 1]
```diff
@@ -783,6 +783,32 @@ Default: `DEFAULTS`
...
 
+##### --ssl_protocols
+
+Specifies an explicit allow-list of TLS protocols for YugabyteDB's internal RPC communication.
+
+Default: An empty string, which is equivalent to allowing all protocols except "ssl2" and "ssl3".
+
+You can pass a comma-separated list of strings, where the strings can be one of "ssl2", "ssl3", "tls10", "tls11", "tls12", and "tls13".
+
+You can set the TLS version for node-to-node and client-node communication. To enforce TLS 1.2, set the flag to tsl12 as follows:
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-tserver.md:794]:
```suggestion
You can set the TLS version for node-to-node and client-node communication. To enforce TLS 1.2, set the flag to tls12 as follows:
```

File: docs/content/stable/reference/configuration/yb-tserver.md [hunk 2]
```diff
@@ -846,6 +846,32 @@ Default: `DEFAULTS`
...
 
+##### --ssl_protocols
+
+Specifies an explicit allow-list of TLS protocols for YugabyteDB's internal RPC communication.
+
+Default: An empty string, which is equivalent to allowing all protocols except "ssl2" and "ssl3".
+
+You can pass a comma-separated list of strings, where the strings can be one of "ssl2", "ssl3", "tls10", "tls11", "tls12", and "tls13".
+
+You can set the TLS version for node-to-node and client-node communication. To enforce TLS 1.2, set the flag to tsl12 as follows:
```
Comment (ddhodge) [inline docs/content/stable/reference/configuration/yb-tserver.md:857]: tls

File: docs/content/v2.8/reference/configuration/yb-tserver.md [hunk 3]
```diff
@@ -804,6 +816,32 @@ Default: `DEFAULTS`
...
 
+##### --ssl_protocols
+
+Specifies an explicit allow-list of TLS protocols for YugabyteDB's internal RPC communication.
+
+Default: An empty string, which is equivalent to allowing all protocols except "ssl2" and "ssl3".
+
+You can pass a comma-separated list of strings, where the strings can be one of "ssl2", "ssl3", "tls10", "tls11", "tls12", and "tls13".
+
+You can set the TLS version for node-to-node and client-node communication. To enforce TLS 1.2, set the flag to tsl12 as follows:
```
Comment (ddhodge) [inline docs/content/v2.8/reference/configuration/yb-tserver.md:827]: tls

File: docs/content/preview/reference/configuration/yb-tserver.md [hunk 4]
```diff
@@ -783,6 +783,32 @@ Default: `DEFAULTS`
...
+--ssl_protocols = tls12
+```
+
+To specify a _minimum_ TLS version of 1.2, for example, the flag needs to be set to tls12, tls13, and all available subsequent versions.
+
+```sh
+--ssl_protocols = tls12,tls13
+```
+
+In addition, as this setting does not propagate to PostgreSQL, it is recommended that you specify the minimum TLS version ( `ssl_min_protocol_version` ) for PostgreSQL by setting the [`ysql_pg_conf_csv`](#ysql-pg-conf-csv) flag as follows:
```
Comment (ddhodge) [inline docs/content/preview/reference/configuration/yb-tserver.md:806]: extra spaces in parentheses?

**Review by ddhodge — APPROVED** [review]
LGTM

**Pull request merged**